cmd/kubeseal-convert/secretsmanager.go

```diff
@@ -11,11 +11,14 @@ var secretsmanagerCmd = &cobra.Command{
 	Short:   "Convert AWS Secrets-Manager secrets",
 	Args:    cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		secretData := kubesealconvert.Secretsmanager(args[0], cmd)
-
-		// Build Sealed Secrets using AWS Secrets Manager data
-		kubesealconvert.BuildSecretFile(secretData)
-
+		secretVal := domain.SecretValues{
```

Comment:
this change keeps all the "cli" "cobra" logic in the cmd  package and not vie versa